desktop/app/installer.js

```diff
@@ -0,0 +1,36 @@
+export default (callback) => {
+  const app = require('electron').app;
+  const nslog = require('nslog');
+  const exec = require('child_process').exec;
+  const path = require("path");
+  const fs = require('fs');
+
+  // const exePath = app.getPath("exe")
```

Comment:
Maybe remove the commented-out lines or parameterize on debug?  We have a `__DEV__` but might not be set this early.